a3p-integration/package.json

```diff
@@ -11,6 +11,7 @@
     },
     "dependencies": {
         "@agoric/synthetic-chain": "^0.0.4-3",
+        "agoric": "^0.22.0-u13.0",
```

Comment:
Oh yeah, why are we depending on the published version here?

Reply:
aye, this was to get the version when it was in a separate workspace. I put into the SDK workspace, which makes this change unnecessary (and harmful)

Reply:
I had to revert because mixing a3p-integration into the root project ran into too many assumptions made about workspace packages